README.md

```diff
@@ -2,40 +2,60 @@
 
 ## Windows
 
-*TODO*: Write up Windows build instructions based off of what's in release-windows.yml
+_TODO_: Write up Windows build instructions based off of what's in release-windows.yml
 
 ## Linux
 
 - Install dependencies using apt:
+
 ```shell
-sudo apt-get install git build-essential zip libkrb5-dev nasm graphviz python3 python3-dev autoconf libfontconfig1-dev gperf libxslt1-dev xsltproc libxml2-utils libxrandr-dev libx11-dev bison flex libgtk-3-dev libgstreamer-plugins-base1.0-dev libgstreamer1.0-dev libnss3-dev libavahi-client-dev libxt-dev 
+sudo apt-get install git build-essential zip libkrb5-dev nasm graphviz python3 python3-dev autoconf libfontconfig1-dev gperf libxslt1-dev xsltproc libxml2-utils libxrandr-dev libx11-dev bison flex libgtk-3-dev libgstreamer-plugins-base1.0-dev libgstreamer1.0-dev libnss3-dev libavahi-client-dev libxt-dev
 ```
 
 ## macOS
+
 - [Download and install Xcode 13.4.1](https://developer.apple.com/services-account/download?path=/Developer_Tools/Xcode_13.4.1/Xcode_13.4.1.xip)
   - Open Xcode after installing and agree to the EULA
-- [Install brew](https://brew.sh)
-- Install dependencies using brew:
-``` shell
-brew install nasm autoconf automake gperf
+  - If you are on MacOS Ventura or later you will be unable to launch this version of Xcode normally and will need to launch it via the terminal and update your `DEVELOPER_DIR` accordingly
+
+```shell
+open /Applications/Xcode13.4.1.app/Contents/MacOS/Xcode
+export DEVELOPER_DIR=/Applications/Xcode13.4.1.app/Contents/Developer
+```
+
+- [Install MacPorts](https://www.macports.org/install.php)
+- Install dependencies using MacPorts:
+
+```shell
+sudo port install gperf
+sudo port install gpatch
+sudo port install nasm
+sudo port install autoconf
+sudo port install automake
```

Comment:
You can do `port install gperf gpatch ...`, multiple packages on one line

Reply:
Updated doc